src/clover/analysis.py

```diff
@@ -102,12 +102,13 @@ def get_key_results(
     key_results.average_pv_generation = float(
         round(total_solar_generation / (20 * 365))
     )
-
+    
     # Compute the grid results.
-    if grid_input_profile is not None:
-        key_results.grid_daily_hours = np.sum(
-            grid_input_profile[: num_years * HOURS_PER_YEAR], axis=0
-        ) / (365 * num_years)
+    for grid_profile in grid_profiles:
```

Comment:
@BenWinchester Can you please check that and tell me if it is correct (Got a bit confused with the grid_input...